### PR TITLE
Legal stuff links in footer

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: core/src/luigi-config/nav-footer.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { getBusolaClusterParams } from './busola-cluster-params';
+
+// setting sideNavFooterText of Luigi settings won't allow HTML
+export async function setNavFooterText() {
+  const targetElement = document.querySelector('.fd-side-nav');
+
+  // we can't add footer if left nav is hidden
+  if (!targetElement) return;
+
+  // the footer is already here
```
Comment (dariadomagala) [inline core/src/luigi-config/nav-footer.js:11]: Can't we just set the footer text as empty at the beginning to avoid it?

**Review by dariadomagala — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -232,16 +231,20 @@ clusters:
     title-current-cluster: Cluster Overview
     version: Version
   storage:
-    changed-message: The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to {{targetStorage}}.
+    changed-message: >-
+      The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:235]:
```suggestion
      The chosen storage type has been overwritten by the cluster configuration from {{originalStorage}} to {{targetStorage}}.
```

Why split that sentence to two lines? Not good for translators.

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -232,16 +231,20 @@ clusters:
...
+      {{targetStorage}}.
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:236]:
```suggestion
```

File: core/src/i18n/en.yaml [hunk 4]
```diff
@@ -232,16 +231,20 @@ clusters:
...
     choose-storage:
       label: Storage type configuration
     descriptions:
       inMemory: Cluster data is cleared when you leave or refresh the page.
       localStorage: Cluster data is persisted between browser reloads.
       sessionStorage: Cluster data is cleared when the page session ends.
       unknown: Unknown storage type
-    info: |
-      Now you can choose where we store your cluster configuration. Based on your kubeconfig type, we recomment the pre-selected storage mode. Of course, you can choose whateer you prefer for convenience and security. None of the options stores information on the server.
+    info: >
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:244]: No. Don't split sentences, that can't be good for translation.

File: core/src/i18n/en.yaml [hunk 5]
```diff
@@ -901,6 +943,11 @@ jobs:
   title: Jobs
 kubeconfig-id:
   error: "Couldn't load kubeconfig ID; configuration not changed (Error: ${{error}})."
+legal:
```
Comment (NHingerl) [inline core/src/i18n/en.yaml:946]: Legal sections are alright. 

But I won't approve unless we know for sure that the translation workflow can accommodate split sentences

**Review by NHingerl — CHANGES_REQUESTED** [review]

File: core/src/i18n/en.yaml (hunk 5 above)
Comment (NHingerl) [reply core/src/i18n/en.yaml:903]: sentences were combined again :)

**Review by NHingerl — APPROVED** [review]

**Pull request merged**